.circleci/config.yml

```diff
@@ -443,19 +443,26 @@ commands:
     parameters:
       test_plan:
         type: string
+      scheme:
+        type: string
+        default: "BackendIntegrationTests"
+      bundle_name:
+        type: string
+        default: "BackendIntegrationTests"
     steps:
       - checkout
       - install-dependencies
+      - tuist-generate-workspace
       - update-spm-installation-commit
       - run:
-          name: Run backend_integration Tests
-          command: bundle exec fastlane backend_integration_tests test_plan:"<< parameters.test_plan >>"
+          name: Run backend_integration Tests with Tuist
+          command: bundle exec fastlane backend_integration_tests test_plan:"<< parameters.test_plan >>" scheme:"<< parameters.scheme >>"
           no_output_timeout: 5m
           environment:
-            SCAN_DEVICE: iPhone 16 (18.0.0)
+            SCAN_DEVICE: iPhone 16 (18.5)
```

Comment:
This is probably a good idea since 18.5 is more commonly used than 18.0 nowadays. But do we (have to) maintain a list somewhere of iOS versions we test on for the iOS majors we support? Or is 'any' version per major fine?

Reply:
That's a good point. I don't think we have a list for this, although it would be great to have it. I changed this one because https://circle-macos-docs.s3.amazonaws.com/image-manifest/v15338/manifest.txt only has 18.5 or 17.5 and I didn't want to downgrade.